manifests/t/TorProject/TorBrowserAlpha/11.5a9/TorProject.TorBrowserAlpha.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: TorProject.TorBrowserAlpha

```

```suggestion
PackageIdentifier: TorProject.TorBrowser.Alpha
```
Since this is a different channel of an already existing package, it should be put under the existing package.